README.md

```diff
@@ -7,3 +7,38 @@ Connector for BI](https://docs.mongodb.com/bi-connector/current/),
 please see the
 [mongodb/mongo-bi-connector-odbc-driver](https://github.com/mongodb/mongo-bi-connector-odbc-driver)
 repository.
+
+
+## Set up the ODBC driver on Windows
+Note: users can utilize the built-in driver manager.
+1. Update the values of `Driver`, `Pwd`, `Server`, `User`, and `Database` in `setup/setupDSN.reg`. The value of `Driver` should be the absolute path of `mongoodbc.dll`. This file should be located in either the `mongo-odbc-driver/target/debug` directory or in the release directory.
+
+2. For 32-bit architectures, use the file path `HKEY_LOCAL_MACHINE\Wow6432Node\SOFTWARE` instead of `HKEY_LOCAL_MACHINE\SOFTWARE` in `setupDSN.reg`.
```

Comment:
```suggestion
2. For 32-bit architectures, update the file path to `HKEY_LOCAL_MACHINE\Wow6432Node\SOFTWARE` instead of `HKEY_LOCAL_MACHINE\SOFTWARE` in `setupDSN.reg`.
```